test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1152,6 +1167,34 @@ def forward(self, input):
         x = torch.randn(3, 4, 5)
         self.run_test(UnbindModel2(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
```

Comment:
Maybe adding a couple of tests for negative indices for getitem and unbind? ONNX does support negative indices so it should be fine.

Reply:
Good point, added